app/services/notifications.rb

```diff
@@ -51,4 +54,20 @@ def self.organization_data(organization)
       profile_image_90: organization.profile_image_90
     }
   end
+
+  def self.ancestor_data(comment)
```

Comment:
I tried to make this a private method but it didn't work. It's technically only called within the `Notifications` module, so IMO it doesn't need to be a `self.` method, but also not 100% sure how modules work. 😬

Reply:
I'm not 100% sure either, Ruby modules are weird :D I just remember you have to put `private_class_method :hello` after the method, so something like

```ruby
module M
  def self.hello
     puts "hello M"
  end
  private_class_method :hello
end
```

Reply:
Nice, looks like it works! 🙌